lib/twine/stringsfile.rb

```diff
@@ -48,24 +48,12 @@ def matches_tags?(tags, include_untagged)
       return false
     end
 
-    def translated_string_for_lang(lang, default_lang=nil)
+    def translated_string_for_lang(lang)
       translation = [lang].flatten.map { |l| @translations[l] }.first
 
-      translation = reference.translated_string_for_lang(lang, default_lang) if translation.nil? && reference
+      translation = reference.translated_string_for_lang(lang) if translation.nil? && reference
 
       return translation if translation
```

Comment:
With the code after this gone, I think we should explicitly return something. Probably translation if set else nil.